.golangci.yml

```diff
@@ -54,21 +54,25 @@ issues:
 
 linters-settings:
   misspell:
-    # Correct spellings using locale preferences for US or UK.
```

Comment:
Sorry I didn't mean all of them, these are useful.
I meant the big block below that wasn't used.

Reply:
I'll re-add in a later PR working towards #15546